### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by DanLong** [SanderSade/DirLister#5]
I love this.  
I have been using Treeprint for years, and found this when Treeprint did not work on Windows 10 64 bit (not a bad run, 19 years for a simple directory listing utility.)

Small feature request:  Is it possible to add an option to list just the directory tree, not the files?  Or is this already possible with the existing version, and I just haven't been able to figure out how?
Thanks,
Dan

**Comment by SanderSade** [comment]
Sorry for the delayed reply - RL happens. Thank you for the kind words!

Yes, this is easily doable. I have some updates planned, but almost certainly not before end of November, and possibly even later.

**Comment by DanLong** [comment]
Related question:  Is it possible to add a toggle to include empty folders in the directory listing results?

**Comment by SanderSade** [comment]
Yes, that is a minor enhancement. It might be config-file-only option, though.

**Comment by DanLong** [comment]
That would be fine, for me.  :)  Thanks!
